azkaban-common/src/main/java/azkaban/project/JdbcProjectLoader.java

```diff
@@ -116,6 +132,38 @@ public JdbcProjectLoader(Props props) {
     return projects;
   }
 
+  private List<Project> fetchAllInactiveProjects(Connection connection)
+          throws ProjectManagerException {
+    QueryRunner runner = new QueryRunner();
+
+    ProjectResultHandler handler = new ProjectResultHandler();
+    List<Project> projects = null;
+    try {
+      projects =
+              runner.query(connection,
+                      ProjectResultHandler.SELECT_ALL_INACTIVE_PROJECTS, handler);
+
+      for (Project project : projects) {
```

Comment:
Common code w.r.t to fetchAllActiveProjects, can we avoid it

Reply:
fetchAllActiveProjects was getting used in other flows like populating that internal cache and which seemed critical to me, did not want to impact that flow. Can I add a TODO for this for now?